pkgs/applications/misc/comfyui/default.nix

```diff
@@ -0,0 +1,93 @@
+{ lib
+, python311
```

Comment:
If you don't specifically need 3.11, just use `python3` for the latest version.

Reply:
according to [upstream docs](https://github.com/comfyanonymous/ComfyUI#manual-install-windows-linux), torch does not work on python 3.12 yet